web/vapour-style.json

```diff
@@ -254,47 +254,97 @@
       }
     },
     {
-      "id": "transport_ways_filtered_outline",
-      "type": "line",
+      "id": "line-intercity-copy",
```

Comment:
We need better names here too :grin: 
`line-intercity-shields` ?

Reply:
Right, fixed in new commit.